### PR TITLE
Update console.py

### PR DESCRIPTION
**Pull request by azzaz3kl**
without adding a new branch
Let's adding a new pull request for this changes and view it in the other account to check changes after making fork from the main repo

### CODE REVIEW

**Review by 3zzazakl — COMMENTED** [review]
Thanks for making fork  for this repo

**Comment by azzaz3kl** [comment]
we did it

**Comment by 3zzazakl** [comment]
yeh

**Review by azzaz3kl — COMMENTED** [review]
done

**Comment by 3zzazakl** [comment]
ok

**Review by 3zzazakl — COMMENTED** [review]
done

**Review by azzaz3kl — COMMENTED** [review]
done

**Review by azzaz3kl — COMMENTED** [review]
done

File: console.py [hunk 1]
```diff
@@ -3,3 +3,7 @@
 
 print("this is file is created by 3zzazakl want to see the chanegs after making fork and pull requests")
 print("Testing another pull request")
+
+print("Adding this line after forking this repo into aother account")
+print("Well Done")
+print("Push this changes wihtout adding a new branch!!!")
```
Comment (3zzazakl) [inline console.py:9]: done

**Review by 3zzazakl — COMMENTED** [review]
done

File: console.py (hunk 1 above)
Comment (3zzazakl) [inline console.py:9]: done

**Review by 3zzazakl — COMMENTED** [review]
done

**Pull request merged**